tests/jest.config.ts

```diff
@@ -14,14 +14,14 @@ export interface ITestConfig {
 
 export const options: ITestConfig = {
   loginOptions: {
-    cookieA: "cc08dd90-c453-49c7-8144-2e6ffe1197d7", // your cookies
+    cookieA: "c0620986-f48a-4bd7-971c-2a25566e0a7d", // your cookies
```

Comment:
I logged out the user to invalidate the token, otherwise people are able to sue your account.